### ISSUE TITLE
iCal feed of notes for action

### ISSUE THREAD

**Issue by tbar0970** [tbar0970/jethro-pmm#124]
IN Jethro you can assign a note for action from a certain date.
It would be handy to have an iCal feed that can show these in your calendar.
Need to investigate: I think there is some iCal extension that represents tasks.  Not sure how widely supported it is.

**Comment by tbar0970** [comment]
@eb3nezer any interest in having a look at this?

**Comment by jameskirsop** [comment]
FWIW, the [iCalendar IETF spec](https://www.ietf.org/rfc/rfc2445.txt) is actually pretty basic, and it might not be too much effort to implement this without a third party library. I haven't taken a look at the Jethro source for this, though.

**Comment by tbar0970** [comment]
Yep, shouldn't need any libraries.  But I expect it would need careful testing for support in different calendar clients.
